cypress/integration/edit-route/info-link/index.js

```diff
@@ -0,0 +1,23 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const infoHref =
```

Comment:
I guess this is off-topic to this this PR but this link redirects to https://docs.dhis2.org/en/home.html :disappointed:

Reply:
Yeah exactly. Phil has merged the docs PR here: https://github.com/dhis2/dhis2-docs/pull/631, so it should all work once 2.36 has been released.